### ISSUE TITLE
GRAPHQL "Internal server error"

### ISSUE THREAD

**Issue by anirudh7** [pods-framework/pods#6865]
### Description

"Call to private Pods\\Pod_Manager::__construct() from context 'tad_DI52_Container'"

### Version

2.9.1

### Screenshots / Screencast

Pods 2.9.1 without Pods Pro WPGRAPHQL Plugin
![Screenshot (413)](https://user-images.githubusercontent.com/5333996/184667219-4907d2f5-b1fc-4b2e-a306-43cd6054d559.png)

### Possible Workaround

Pods 2.9.1 with Pods Pro WPGRAPHQL Plugin
![Web capture_15-8-2022_211350_app itsnz org](https://user-images.githubusercontent.com/5333996/184668260-a9e668b9-f724-4405-adc9-aa4344e0ff76.jpeg)

### Site Health Information

```text
`
### wp-core ###

version: 6.0.1
site_language: en_NZ
user_language: en_NZ
timezone: +12:00
permalink: /%postname%/
https_status: true
multisite: false
user_registration: 0
blog_public: 0
default_comment_status: undefined
environment_type: production
user_count: 1
dotorg_communication: true

### wp-paths-sizes ###

wordpress_path: /var/www/vhosts/itsnz.org/app.itsnz.org
wordpress_size: 56.61 MB (59362417 bytes)
uploads_path: /var/www/vhosts/itsnz.org/app.itsnz.org/wp-content/uploads
uploads_size: 0.00 B (0 bytes)
themes_path: /var/www/vhosts/itsnz.org/app.itsnz.org/wp-content/themes
themes_size: 11.53 MB (12089471 bytes)
plugins_path: /var/www/vhosts/itsnz.org/app.itsnz.org/wp-content/plugins
plugins_size: 44.57 MB (46736481 bytes)
database_size: 8.73 MB (9158656 bytes)
total_size: 121.45 MB (127347025 bytes)

### wp-active-theme ###

name: Twenty Twenty-One (twentytwentyone)
version: 1.6
author: the WordPress team
author_website: https://wordpress.org/
parent_theme: none
theme_features: core-block-patterns, widgets-block-editor, automatic-feed-links, title-tag, post-formats, post-thumbnails, menus, html5, custom-logo, customize-selective-refresh-widgets, wp-block-styles, align-wide, editor-styles, editor-style, editor-font-sizes, custom-background, editor-color-palette, editor-gradient-presets, responsive-embeds, custom-line-height, experimental-link-color, custom-spacing, custom-units, widgets
theme_path: /var/www/vhosts/itsnz.org/app.itsnz.org/wp-content/themes/twentytwentyone
auto_update: Disabled

### wp-themes-inactive (2) ###

Twenty Twenty: version: 2.0, author: the WordPress team, Auto-updates disabled
Twenty Twenty-Two: version: 1.2, author: the WordPress team, Auto-updates disabled

### wp-plugins-active (4) ###

Gravity Forms: version: 2.6.5, author: Gravity Forms, Auto-updates disabled
Pods - Custom Content Types and Fields: version: 2.9.1, author: Pods Framework Team, Auto-updates disabled
Pods Pro - WPGraphQL Add-On: version: 1.0.1, author: SKC Development, LLC, Auto-updates disabled
WP GraphQL: version: 1.9.0, author: WPGraphQL, Auto-updates disabled

### wp-plugins-inactive (3) ###

Akismet Anti-Spam: version: 5.0, author: Automattic, Auto-updates disabled
Pods Gravity Forms Add-On: version: 1.4.5, author: Pods Framework Team, Auto-updates disabled
Wordfence Security: version: 7.6.0, author: Wordfence, Auto-updates disabled

### wp-media ###

image_editor: WP_Image_Editor_Imagick
imagick_module_version: 1690
imagemagick_version: ImageMagick 6.9.10-68 Q16 x86_64 2021-10-14 https://imagemagick.org
imagick_version: 3.7.0
file_uploads: File uploads is turned off
post_max_size: 8M
upload_max_filesize: 4M
max_effective_size: 4 MB
max_file_uploads: 20
imagick_limits: 
	imagick::RESOURCETYPE_AREA: 4 GB
	imagick::RESOURCETYPE_DISK: 9.2233720368548E+18
	imagick::RESOURCETYPE_FILE: 6144
	imagick::RESOURCETYPE_MAP: 4 GB
	imagick::RESOURCETYPE_MEMORY: 2 GB
	imagick::RESOURCETYPE_THREAD: 1
imagemagick_file_formats: 3FR, 3G2, 3GP, AAI, AI, ART, ARW, AVI, AVS, BGR, BGRA, BGRO, BMP, BMP2, BMP3, BRF, CAL, CALS, CANVAS, CAPTION, CIN, CIP, CLIP, CMYK, CMYKA, CR2, CRW, CUR, CUT, DATA, DCM, DCR, DCX, DDS, DFONT, DNG, DOT, DPX, DXT1, DXT5, EPDF, EPI, EPS, EPS2, EPS3, EPSF, EPSI, EPT, EPT2, EPT3, ERF, EXR, FAX, FILE, FITS, FRACTAL, FTP, FTS, G3, G4, GIF, GIF87, GRADIENT, GRAY, GRAYA, GROUP4, GV, H, HALD, HDR, HISTOGRAM, HRZ, HTM, HTML, HTTP, HTTPS, ICB, ICO, ICON, IIQ, INFO, INLINE, IPL, ISOBRL, ISOBRL6, J2C, J2K, JNG, JNX, JP2, JPC, JPE, JPEG, JPG, JPM, JPS, JPT, JSON, K25, KDC, LABEL, M2V, M4V, MAC, MAGICK, MAP, MASK, MAT, MATTE, MEF, MIFF, MKV, MNG, MONO, MOV, MP4, MPC, MPEG, MPG, MRW, MSL, MSVG, MTV, MVG, NEF, NRW, NULL, ORF, OTB, OTF, PAL, PALM, PAM, PANGO, PATTERN, PBM, PCD, PCDS, PCL, PCT, PCX, PDB, PDF, PDFA, PEF, PES, PFA, PFB, PFM, PGM, PGX, PICON, PICT, PIX, PJPEG, PLASMA, PNG, PNG00, PNG24, PNG32, PNG48, PNG64, PNG8, PNM, PPM, PREVIEW, PS, PS2, PS3, PSB, PSD, PTIF, PWP, RADIAL-GRADIENT, RAF, RAS, RAW, RGB, RGBA, RGBO, RGF, RLA, RLE, RMF, RW2, SCR, SCT, SFW, SGI, SHTML, SIX, SIXEL, SPARSE-COLOR, SR2, SRF, STEGANO, SUN, SVG, SVGZ, TEXT, TGA, THUMBNAIL, TIFF, TIFF64, TILE, TIM, TTC, TTF, TXT, UBRL, UBRL6, UIL, UYVY, VDA, VICAR, VID, VIFF, VIPS, VST, WBMP, WMF, WMV, WMZ, WPG, X, X3F, XBM, XC, XCF, XPM, XPS, XV, XWD, YCbCr, YCbCrA, YUV
gd_version: bundled (2.1.0 compatible)
gd_formats: GIF, JPEG, PNG, WebP, BMP, XPM
ghostscript_version: not available

### wp-server ###

server_architecture: Linux 3.10.0-1127.18.2.el7.x86_64 x86_64
httpd_software: Apache
php_version: 7.4.30 64bit
php_sapi: fpm-fcgi
max_input_variables: 1000
time_limit: 30
memory_limit: 128M
admin_memory_limit: 256M
max_input_time: 60
upload_max_filesize: 4M
php_post_max_size: 8M
curl_version: 7.29.0 NSS/3.53.1
suhosin: false
imagick_availability: true
pretty_permalinks: true
htaccess_extra_rules: false

### wp-database ###

extension: mysqli
server_version: 5.5.68-MariaDB
client_version: mysqlnd 7.4.30
max_allowed_packet: 1048576
max_connections: 151

### wp-constants ###

WP_HOME: undefined
WP_SITEURL: undefined
WP_CONTENT_DIR: /var/www/vhosts/itsnz.org/app.itsnz.org/wp-content
WP_PLUGIN_DIR: /var/www/vhosts/itsnz.org/app.itsnz.org/wp-content/plugins
WP_MEMORY_LIMIT: 40M
WP_MAX_MEMORY_LIMIT: 256M
WP_DEBUG: false
WP_DEBUG_DISPLAY: true
WP_DEBUG_LOG: false
SCRIPT_DEBUG: false
WP_CACHE: false
CONCATENATE_SCRIPTS: undefined
COMPRESS_SCRIPTS: undefined
COMPRESS_CSS: undefined
WP_ENVIRONMENT_TYPE: Undefined
DB_CHARSET: utf8
DB_COLLATE: undefined

### wp-filesystem ###

wordpress: writable
wp-content: writable
uploads: writable
plugins: writable
themes: writable

### pods ###

pods-server-software: Apache
pods-user-agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/104.0.5112.81 Safari/537.36 Edg/104.0.1293.54
pods-session-save-path: /tmp
pods-session-save-path-exists: Yes
pods-session-save-path-writable: Yes
pods-session-max-lifetime: 1440
pods-opcode-cache-apc: No
pods-opcode-cache-memcached: No
pods-opcode-cache-opcache: No
pods-opcode-cache-redis: No
pods-object-cache-apc: No
pods-object-cache-apcu: No
pods-object-cache-memcache: No
pods-object-cache-memcached: No
pods-object-cache-redis: Yes
pods-memory-current-usage: 30.012M
pods-memory-current-usage-real: 18.000M
pods-network-wide: No
pods-install-location: /wp-content/plugins/pods/
pods-developer: No
pods-tableless-mode: No
pods-relationship-table-enabled: Yes
pods-light-mode: No
pods-strict: No
pods-allow-deprecated: Yes
pods-api-cache: Yes
pods-shortcode-allow-evaluate-tags: No
pods-sessions: Disable sessions
pods-can-use-sessions: Yes
pods-relationship-table-status: 0

`
```

### Pods Package

```json
{
    "@meta": {
        "version": "2.9.1",
        "build": 1660578524
    },
    "settings": {
        "types_only": "0",
        "watch_changed_fields": "0",
        "metadata_integration": "1",
        "metadata_override_get": "0",
        "session_auto_start": "0",
        "wisdom_opt_out": "1"
    },
    "pods": [
        {
            "name": "itsnz_speaker",
            "id": 45,
            "label": "ITSNZ Speakers",
            "description": "",
            "type": "post_type",
            "storage": "meta",
            "label_singular": "ITSNZ Speaker",
            "public": "1",
            "show_ui": "1",
            "rest_enable": "1",
            "supports_title": "1",
            "supports_editor": "1",
            "_migrated_28": "1",
            "required": "0",
            "repeatable": "0",
            "meta_box_context": "normal",
            "meta_box_priority": "default",
            "wpgraphql_enabled": "1",
            "wpgraphql_all_fields_enabled": "1",
            "wpgraphql_pick_format": "connection",
            "wpgraphql_file_format": "connection",
            "write_all": "itsnz_speaker",
            "pods_pro_wpgraphql_enabled": "1",
            "pods_pro_wpgraphql_all_fields_enabled": "1",
            "groups": [
                {
                    "name": "more_fields",
                    "id": 46,
                    "label": "More Fields",
                    "description": "",
                    "weight": 0,
                    "fields": [
                        {
                            "name": "speaker_name",
                            "id": 47,
                            "label": "Speaker Name",
                            "description": "",
                            "weight": 0,
                            "type": "text",
                            "sister_id": "-- Select One --",
                            "required": "0",
                            "repeatable": "0",
                            "text_allowed_html_tags": "strong em a ul ol li b i",
                            "text_max_length": "255",
                            "roles_allowed": "administrator",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "1",
                            "wpgraphql_enabled": "1",
                            "wpgraphql_pick_format": "connection",
                            "wpgraphql_file_format": "connection",
                            "text_trim": "1",
                            "pods_pro_wpgraphql_enabled": "1",
                            "pods_pro_wpgraphql_pick_format": "connection",
                            "pods_pro_wpgraphql_file_format": "connection"
                        }
                    ]
                }
            ]
        }
    ]
}
```

**Comment by JoryHogeveen** [comment]
Hi @anirudh7 

> "Call to private Pods\Pod_Manager::__construct() from context 'tad_DI52_Container'"

Can you provide us with a full backtrace stack?

**Comment by anirudh7** [comment]
> Can you provide us with a full backtrace stack?

Sorry! Can I know how should I achieve this? what will be the recommended tools?

**Comment by anirudh7** [comment]
Also I have done a fresh installation of WordPress with only two plugins enabled (WPGRAPHQL and PODS 2.9.2) but still got the same error.

**Comment by sc0ttkclark** [comment]
Fixed via 0b071d873deef981904a726b9bcb682d7d24e39c

We didn't need a private __construct() here anyways, it was leftover from something else.

**Comment by anirudh7** [comment]
New Error after updating to 2.9.2

`{
  "errors": [
    {
      "debugMessage": "Access to undeclared static property: Pods\\Pod_Manager::$pods",
      "message": "Internal server error",
      "extensions": {
        "category": "internal"
      },
      "locations": [
        {
          "line": 5,
          "column": 9
        }
      ],
      "path": [
        "itsnzSpeakers",
        "edges",
        0,
        "node",
        "speakerName"
      ]
    }
  ],
  "data": {
    "itsnzSpeakers": {
      "edges": [
        {
          "node": {
            "speakerName": null
          }
        }
      ]
    }
  },
  "extensions": {
    "debug": []
  }
}`
